GOVERNANCE.md

```diff
@@ -0,0 +1,135 @@
+### io.js Build Team Working Group
+
+The io.js build team is jointly governed by a Working Group (WG)
+which is responsible for high-level guidance of the project.
+
+The WG has final authority over this project including:
+
+* Technical direction
+* Project governance and process (including this policy)
+* Contribution policy
+* GitHub repository hosting
+* Conduct guidelines
+* Maintaining the list of additional Collaborators
+
+For the current list of WG members, see the project
+[README.md](./README.md#current-project-team-members).
```

Comment:
Agreed. What should we do about the current section: https://github.com/iojs/build#people